safedata_validator/zenodo.py

```diff
@@ -91,12 +93,12 @@ def _resources_to_zenodo_api(resources: Optional[Resources] = None) -> dict:
         contact_affiliation = resources.zenodo.contact_affiliation
         contact_orcid = resources.zenodo.contact_orcid
 
-    # Get the metadata api
-    metadata_api = resources.metadata.api
-    metadata_token = resources.metadata.token
-    if metadata_api is None or metadata_token is None:
-        config_fail = True
-    metadata_ssl = resources.metadata.ssl_verify
+    # # Get the metadata api
+    # metadata_api = resources.metadata.api
+    # metadata_token = resources.metadata.token
+    # if metadata_api is None or metadata_token is None:
+    #     config_fail = True
+    # metadata_ssl = resources.metadata.ssl_verify
```

Comment:
Is the above commented out because the metadata server isn't setup yet?

Reply:
It's a wider flow issue. The previous setup generated Gemini 2 XML on the metadata server and inserted a link to that API call into the Zenodo description. That means that the Zenodo publication step requires a metadata server to be active to generate the Zenodo description, which is why that code was in there.

I'm regretting that, because it is cleaner if `sdv` can be used with the hierarchy:  validate, publish, discovery. Users should only have to spin up a metadata server if they want the discovery and `safedata` in R and having validate and publish as stand alone things is really useful.

So - we'd like backwards compatibility with those links (and the metadata server _can_ generate Gemini XML easily enough) but I think what we really want is for `sdv 3.0.0` to recommend adding the Gemini XML as part of the Zenodo record. However we do want the Zenodo description to point to the metadata server if one _is_ available.

I haven't thought this through, but this bit needs resolving.